### DISCUSSION TITLE
Url fields in documents

### DISCUSSION THREAD

**Discussion by AKuederle** [oramasearch/orama#444] [General]
I am currently playing around with Orama as search provider for a database that has URLs as some fields and free text in others.

Do to the way the default stemming and word splitting works the results for the URL fields don't seem to work very well.
For example, if I search for the exact URL of one items, it will often not get returned as the top match, but some partial match in the free text of some other item will get a higher score.

Does anyone has a some recommendations on how to tweak the settings for the URL fields to get better results?

**Comment by allevo** [comment]
Hi!
Thank you for opening this issue.

Currently, Orama doesn't support "URL" type.
For now, we were focused on strings, numbers, and boolean (or array of them) where we needed stemming typo tolerance.

Is the example you explained a real use case?

**Reply by AKuederle** [reply]
Yes this is based on a real-world usecase I was working on. Basically, I am building a small PDF search engine, where users can add PDFs by URL.
At the moment I have the URL as one of the searchable fields, as I expect users to maybe search for the website they downloaded the website from. However, at the moment this is not working perfectly, as special characters within the URL or the searchterm don't give the expected match.
